generator/src/main/java/com/scottlogic/deg/generator/fieldspecs/FieldSpec.java

```diff
@@ -79,16 +80,16 @@ public DateTimeRestrictions getDateTimeRestrictions() {
         return restrictions.get(DateTimeRestrictions.class).orElse(null);
     }
 
-    public FieldSpec withWhitelist(DistributedSet<Object> whitelist) {
-        return new FieldSpec(whitelist, new HeterogeneousTypeContainer<>(), nullable);
+    public FieldSpec withWhitelist(DistributedSet<Object> whitelist) {//todo remove blacklist
```

Comment:
Is this TODO in scope of this PR?

Reply:
Removed the todo, 

blacklist gets removed during fieldspec merging.

we should probably not allow 'with X restriction' in the other than during merging /shrug